content/lifecycle.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lifecycle badges
+---
+
+![](/images/lifecycle/lifecycle.png)
+
+### Experimental ![](https://img.shields.io/badge/lifecycle-experimental-orange.svg) {#experimental}
+
+Very early days, a lot of churn in search of a good API.  Not on CRAN. Might not go anywhere. Use with care.
+
+### Maturing ![](https://img.shields.io/badge/lifecycle-maturing-blue.svg) {#maturing}
+
+API roughed out, but finer details likely to change. Will strive to maintain backward compatibility, but need wider usage in order to get more feedback.
```

Comment:
The API of a maturing package has been roughed out, but finer details likely to change. Once released to CRAN, we will strive to maintain backward compatibility, but the package needs wider usage in order to get more feedback and find the optimal API.